db/migrate/20210723011211_add_cluster_index_sequence.rb

```diff
@@ -0,0 +1,14 @@
+class AddClusterIndexSequence < ActiveRecord::Migration[5.2]
+  def up
+    execute <<-SQL
+      CREATE SEQUENCE cluster_index_seq;
+      SELECT setval('cluster_index_seq', 1000 + 4 * (COUNT(*) + 1)) FROM helm_infrastructures;
```

Comment:
any consideration of using this number?

Reply:
this is starting seq number, we just wanna make sure the starting number is bigger enough than total existing helm_infras. please add more if any mas @raeoks

Reply:
And `1000` was existing padding/offset we were already using. We were looking for a number 2-4x relative to `helm_infrastructures.count` but less than 5000. For 5000 above integers `Rufus::Mnemo.from_integer` adds 1 more syllable to the name. 

e.g.,
```
Rufus::Mnemo.from_i(5000)
=> "bebema"
```